dataplane/loader/Cargo.toml

```diff
@@ -10,11 +10,11 @@ name = "loader"
 path = "src/main.rs"
 
 [dependencies]
+aya = { version = "0.13.1" , features=["async_tokio"] }
+aya-log = { version = "0.2.1" }
```

Comment:
Now I'm kinda second guessing: was there some specific problem with having aya depds defined at the workspace level? Because now we have version configured in multiple places and I'm just curious what the impetus was

Reply:
its the introduction of the controlplane to the workspace. the problem with keeping aya (and netlink) as a workspace dependency is that they are compiled regardless of whether the package you're compiling uses it. having these deps in the workspace would mean that you can't build the contorlplane on macOS from the workspace root, since these creates are meant only for linux